src/main/java/org/takes/misc/Href.java

```diff
@@ -54,6 +55,11 @@
 public final class Href implements CharSequence {
 
     /**
+     * Trailer slash pattern.
+     */
+    private static final Pattern TRAILER = Pattern.compile("/$");
```

Comment:
@Happy-Neko This is a bad name. Please check: http://www.yegor256.com/2015/03/09/objects-end-with-er.html